client/src/components/SharedComponents/Signup/signup.css

```diff
@@ -0,0 +1,27 @@
+body {
+  text-align: center;
+}
+
+button.signup {
```

Comment:
This is a good way of thinking. However, I would still suggest you use BEM for class naming.